WordPress/src/main/res/values/strings.xml

```diff
@@ -1874,6 +1874,7 @@
     <string name="my_site_bottom_sheet_title">Add new</string>
     <string name="my_site_bottom_sheet_add_post">Blog post</string>
     <string name="my_site_bottom_sheet_add_page">Site page</string>
+    <string name="my_site_bottom_sheet_add_story">Story</string>
```

Comment:
I wish we could add a comment for this one, I have no idea how translators will handle that. And I'm not even sure how to translate it in French (the English word "Story" is used in the French translation of Instagram Stories). Maybe you @aerych you thought about that? @mzorz what would be the Spanish translation?

Reply:
Just for reference, the Instagram documentation in French:

<img width="1046" alt="Screenshot 2020-06-04 at 10 34 45" src="https://user-images.githubusercontent.com/40213/83735146-3ed59280-a650-11ea-952e-3e0b2f511fb9.png">

Reply:
This is such an interesting point you made @maxme! ❤️ In spanish it'd be `Historia` and just checked and that's how they label it on Instagram, Fb, WhatsApp apps.  I think we could probably add the comment (not sure how to get the comment to translators?) so they understand this should be just like “instagram stories” for them to translate. @loremattei @oguzkocer do you know of a way to add such a comment in a way that we can tell translatros something like `Please use the same word other services such as Instagram use for Stories`? 😄  (not joking - I think this is the best indicator of how we want this translated here)

Reply:
AFAIK, Android XML format supports developer's comment: any standard XML comment should be attributed to the next string. 
But, I think that GlotPress doesn't support importing comments from XML files. I checked WPAndroid strings on  `translate.wordpress.org` and I don't see comments where I expected them to be. 
Do you know anything more about this @akirk ?

Reply:
Correct, it looks like GlotPress currently ignores these comments. We can pass on this comment in our hand off to translators, though. I added this for our next week's kit: p4DzFg-1eM-p2

Reply:
Thanks @akirk!

Reply:
Thank you Alex!

Reply:
Awesome, thanks a lot @akirk !

Reply:
question @akirk given the feature won't land in `develop` until a few more  weeks / sprints, should I ping you again once we're approaching ? say as soon as we land it in `develop` for example?

Reply:
Tracking in my todo notes, will do that later 👍